views/confirmation.html

```diff
@@ -20,15 +20,27 @@ <h1 class="govuk-panel__title">Application submitted</h1>
         We have sent a confirmation email to <strong>{{ userEmail }}</strong> which contains your reference number.
       </p>
 
+      {% if who_is_registering == "someone_else" %}
+        <h2 class="govuk-heading-m">What you need to do now</h2>
+        <p class="govuk-body">You must give the application reference number to the UK-regulated agent that completed identity checks. </p>
+
+        <p class="govuk-body">Then the agent needs to complete an <a href="/public/docs/overseas-entity-id-checks-statement.odt" class="govuk-link">Overseas entity ID checks statement</a> and email it to Companies House.</p>
+
+        <p class="govuk-body">They must do this within <strong>{{ workingDays }} working days</strong>. We cannot process this application until we've received the statement.</p>
+      {% endif %}
+
       <h2 class="govuk-heading-m">What happens next</h2>
+      <p class="govuk-body">We'll email you to let you know if your application to register an overseas entity has been accepted or rejected. We'll process the application as soon as possible. However, because this is a new service, we're not yet able to tell you exactly how long this will take.</p>
 
-      <p class="govuk-body">We will send an email to <strong>{{ userEmail }}</strong> within {{ workingDays }} working days to let you know if your application to register an overseas entity has been accepted or rejected.</p>
+      <h3 class="govuk-heading-s">What happens if the application is accepted</h3>
+      <p class="govuk-body">We'll email the Overseas Entity ID to <strong>{{ userEmail }}</strong>. We'll send the notice of registration to <strong>{{ entity.email }}</strong>, which will also include the Overseas Entity ID. This will need to be provided to the Land Registry whenever the overseas entity buys, sells or transfers land or property in the UK. </p>
 
-      <p class="govuk-body">If the application is accepted, the overseas entity and its beneficial owners will be added to the <a href="#">Register of Overseas Entities.</a></p>
+      <p class="govuk-body">The overseas entity and its beneficial owners will be added to the Register of Overseas Entities.</p>
 
-      <p class="govuk-body">If it's rejected, we'll explain what to do next in the email.</p>
+      <h3 class="govuk-heading-s">What happens if the application is rejected</h3>
+      <p class="govuk-body">If it's rejected, we'll email you at <strong>{{ userEmail }}</strong> to explain what to do next. We'll also refund the £X registration fee.</p>
```

Comment:
What do you think to create an env variable for the registration fee, similar to the API?

Reply:
the fee is on the start page as well so if we do the env var, teh story will have to change to test the start page too.